README.md

```diff
@@ -32,7 +29,7 @@ The easiest way to have *MBINCompiler* set up is to download the most recent
 release and register *MBINCompiler* to the path so that it can be picked up
 anywhere by Blender.
 If you already have a version of *MBINCompiler* on your computer, ensure it is
-version **v1.78.0-pre1** or above. This can be found on the [MBINCompiler releases](https://github.com/monkeyman192/MBINCompiler/releases) page.
+version **v2.61.1-pre1** or above. This can be found on the [MBINCompiler releases](https://github.com/monkeyman192/MBINCompiler/releases) page.
```

Comment:
I think we can just have this as something along the lines of needing the latest version, rather than having to update this every time (I didn't realise is still said 1.78!!!)